lib/rouge/lexers/swift.rb

```diff
@@ -73,8 +75,8 @@ class deinit enum convenience extension final func import init internal lazy let
         rule /'(\\.|.)'/, Str::Char
         rule /(\d+\*|\d*\.\d+)(e[+-]?[0-9]+)?/i, Num::Float
         rule /\d+e[+-]?[0-9]+/i, Num::Float
```

Comment:
It's not a big deal but for consistency, I'd suggest consistent use of `\d`:

```ruby
rule /\d+e[+-]?\d+/i, Num::Float
```

Reply:
I don't disagree, but I'd like to investigate further improvements separate to this PR.